### PR TITLE
v0.24: Update Asynchronous Updates guide code sample

### LINKED ISSUE #1260: v0.24: Update Asynchronous Updates error message
On the [asynchronous updates guide](https://docs.meilisearch.com/learn/advanced/asynchronous_updates.html#examples) we show a sample error a user may get when the update fails. This sample is out-of-date with current behaviour.

Code sample on the docs:
```json
{
  "status": "failed",
  "updateId": 1,
  "type": { "name": "DocumentsAddition" },
  "enqueuedAt": "2019-12-07T21:10:07.607581330Z",
  "duration": 0.000048524,
  "processedAt": "2019-12-07T21:10:20.511525620Z",
  "error": "document id is missing"
}
```

Code sample received when adding a document without a primary key via the CLI on v0.24 RC2:
```json
{
  "status": "failed",
  "updateId": 0,
  "type": {
    "name": "DocumentsAddition"
  },
  "error": {
    "message": "The primary key inference process failed because the engine did not find any fields containing `id` substring in their name. If your document identifier does not contain any `id` substring, you can set the primary key of the index.",
    "code": "primary_key_inference_failed",
    "type": "invalid_request",
    "link": "https://docs.meilisearch.com/errors#primary_key_inference_failed"
  },
  "duration": 0,
  "enqueuedAt": "2021-11-11T13:31:12.051786Z",
  "processedAt": "2021-11-11T13:31:12.052899Z"
}
```

## Tasks

- [ ] Update failed update code sample on `learn/advanced/asynchronous_updates.md#L109-117

Thanks to @bidoubiwa for noticing this!

### PR DESCRIPTION
**Pull request by guimachiavelli**
Closes #1260

### CODE REVIEW

File: learn/advanced/asynchronous_updates.md [hunk 1]
```diff
@@ -108,12 +108,19 @@ Had the update failed, the response would have included an error message:
...
 {
   "status": "failed",
-  "updateId": 1,
-  "type": { "name": "DocumentsAddition" },
-  "enqueuedAt": "2019-12-07T21:10:07.607581330Z",
-  "duration": 0.000048524,
-  "processedAt": "2019-12-07T21:10:20.511525620Z",
-  "error": "document id is missing"
+  "updateId": 0,
+  "type": {
```
Comment (maryamsulemani97) [inline learn/advanced/asynchronous_updates.md:112]: This should be in one line 👽

```suggestion
"type": { "name": "DocumentsAddition" },
```

Comment (dichotommy) [reply learn/advanced/asynchronous_updates.md]: Is this right? Doesn't it break @bidoubiwa 's `one key/value per line` [principle](https://github.com/meilisearch/documentation/issues/872#issuecomment-803429279)?

**Review by dichotommy — APPROVED** [review]
🚀

**Comment by guimachiavelli** [comment]
bors merge

**Pull request merged**